code/modules/antagonists/_common/antag_datum.dm

```diff
@@ -197,35 +197,15 @@ GLOBAL_LIST_EMPTY(antagonists)
 /datum/antagonist/custom
 	antagpanel_category = "Custom"
 	show_name_in_check_antagonists = TRUE //They're all different
-	var/datum/team/custom_team
+	///var/datum/team/custom_team - TODO: actually implement teams
```

Comment:
Just cut it entirely if it isn't in use now, it can be added back later easily enough.

Reply:
sure